.github/workflows/snyk-scan.yml

```diff
@@ -0,0 +1,51 @@
+---
+name: snyk vanagon check
+
+on:
+  push:
+  pull_request_target:
+      types: [opened, reopened, edited, synchronize]
```

Comment:
I'm kind of surprised this works... I would have thought we need:
```yaml
on:
  - push
  - pull_request_target:
      types: [opened, reopened, edited, synchronize]
```

What you have evaluates to this:
```
irb(main):004:0> YAML.load(File.read('foo.yaml'))
=> {true=>{"push"=>nil, "pull_request_target"=>{"types"=>["opened", "reopened", "edited", "synchronize"]}}}
```
Compare that to what i've suggested:
```
irb(main):003:0> YAML.load(File.read('foo.yaml'))
=> {true=>["push", {"pull_request_target"=>{"types"=>["opened", "reopened", "edited", "synchronize"]}}]}
```

Reply:
👍  I was actually trying that syntax but github's linter rejects it probably mistakenly, will push that.

Reply:
It fails 😮  https://github.com/puppetlabs/bolt-vanagon/actions/runs/1764029008

Reply:
hmmmm

Reply:
Well if it is working then i guess its ok the way it was. yaml is hard

Reply:
Github has some very slippery syntax, thought there was a fix but that also didn't work https://github.community/t/multiple-event-triggers-with-config/16408/4 - oh well 🤷

Reply:
I believe I can change this to 
`on:
  push:
    branches: '**'
  pull_request_target:
      types: [opened, reopened, edited, synchronize]`
 
 which probably is preferable.

Reply:
Works, changed 🚀